README.md

```diff
@@ -65,28 +65,24 @@ public final class Main {
 
 Then just invoke `gradle runExample` to run the Main example!
 
+### Other projects
+
+Below projects need app configurations stored in JSON format in `config.json` file at location `src/example/config/`.
+
+This configuration file can be downloaded from your application's `Configuration` tab at [developer 
+console](https://app.box.com/developers/console)
+
 #### CreateAppUser
 
 This project will output the user id of enterprise admin and will create a new App User for the enterprise.
 
 To run the project, first provide following in `src/example/java/com/box/sdk/example/CreateAppUser.java`.
-* Client Id: From application's [developer console](https://app.box.com/developers/services).
-* Client Secret: From application's [developer console](https://app.box.com/developers/services).
-* Enterprise Id: From Admin Console [Account Info tab](https://app.box.com/master/settings).
-* Public key Id: From application's [developer console](https://app.box.com/developers/services).
-* Private key file name with path: Corresponding to the public key uploaded in application's [developer console](https://app.box.com/developers/services).
-* Private key password (if any): Password for the private key.
+
 * Name of App User: This will be used as the name of the newly created App User.
```

Comment:
This should be reworded so there isn't a bulleted list with just one item. For example:

To run the project, first provide the name of the app user in `src/example/java/com/box/sdk/example/CreateAppUser.java`.